cpp/src/io/json/json_normalization.cu

```diff
@@ -119,20 +122,25 @@ struct TransduceToNormalizedQuotes {
       return '"';
     }
     // Case when the read symbol is an escape character - the actual translation for \<s> for some
-    // symbol <s> is handled by transitions from SEC. For now, there is no output for this
-    // transition
-    if ((match_id == static_cast<SymbolGroupT>(dfa_symbol_group_id::ESCAPE_CHAR)) &&
-        ((state_id == static_cast<StateT>(dfa_states::TT_SQS)))) {
+    // symbol <s> is handled by transitions from SEC. The same logic applies for the transition from
+    // DEC. For now, there is no output for this transition
+    if ((match_id == static_cast<SymbolGroupT>(dfa_symbol_group_id::ESCAPE_CHAR) &&
+         state_id == static_cast<StateT>(dfa_states::TT_SQS)) ||
+        (match_id == static_cast<SymbolGroupT>(dfa_symbol_group_id::ESCAPE_CHAR) &&
+         state_id == static_cast<StateT>(dfa_states::TT_DQS))) {
```

Comment:
optional nit: simplifies logic a bit. Feel free to ignore, if you prefer the original
```suggestion
    if (match_id == static_cast<SymbolGroupT>(dfa_symbol_group_id::ESCAPE_CHAR) &&
        (state_id == static_cast<StateT>(dfa_states::TT_SQS) ||
        state_id == static_cast<StateT>(dfa_states::TT_DQS))) {
```